earth_enterprise/src/common/compressor.cpp

```diff
@@ -15,12 +15,19 @@
 
 #include <string.h>
 #include <math.h>
-#include <libpng12/png.h>
 #include <utility>
 #include "compressor.h"
 #include <common/khConstants.h>
 #include <common/khStringUtils.h>
 
+#ifdef USING_LIBPNG16
+#include <libpng16/png.h>
+#endif
+
+#ifdef USING_LIBPNG15
+#include <libpng15/png.h>
+#endif
+
```

Comment:
even though i guess we aren't supporting it, it might be better to have something like 
`#if defined USING_LIBPNG16`
`#include <libpng16/png.h>`
`#elif defined USING_LIBPNG15`
`#include <libpng15/png.h>`
`#else`
`#include <libpng12/png.h>`
`#endif`

just something to consider, not a showstopper

Reply:
Thats a fair point. Im thinking we will be unable to support 1.2 due to qt4, but using it as a fail safe isn't a bad idea.